string selected
string command
string type
float32 data
float32[] direction
float32[] location
float32[] polygonal_area
float32[] segment
geometry_msgs/Vector3 circ_area
geometry_msgs/Pose sample